R/categorical_trees.R

```diff
@@ -606,10 +607,12 @@ scale_subtree <- function(dt,
     collapse_interval_cols
   )
   setnames(sum_children_dt, value_cols, agg_value_cols)
+  sum_children_dt[, aggregate_exists := TRUE]
```

Comment:
This change was needed to fix two existing tests we have for testing `missing_dt_severity = 'none'` when scaling so that it preserves the original values and doesn't make the missing values NA.
https://github.com/ihmeuw-demographics/hierarchyUtils/blob/feature/simplify_tests/tests/testthat/test_scale.R#L145-L155
https://github.com/ihmeuw-demographics/hierarchyUtils/blob/feature/simplify_tests/tests/testthat/test_scale.R#L325-L334